Examples/Temperature_Salinity_Diagram.ipynb

```diff
@@ -30,9 +30,11 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.        salt = darray
</pre><p>Is there a point to this renaming beyond trying to make it clear we get an array out of the catalog query, and that array represents the data points we want to look at?</p><p> </p><p>It might be clearer to keep the name as <code>darray</code> until the call to <code>gsw_SA_from_SP</code>, which I think is the more natural place to make the change.   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
happy either way!